util/chunk/chunk.go

```diff
@@ -23,11 +23,18 @@ import (
 	"github.com/pingcap/tidb/types/json"
 )
 
+// Sel indicates which rows are selected in a Chunk.
+type Sel []uint16
```

Comment:
I think we can directly use `[]int16`, no need to do a type alias.

Reply:
`[]uint16` or `[]int16`?

Reply:
How about `[]int`? It's easy to use.

Reply:
Okay, updated.